plugins/woocommerce/changelog/42751-update-42072-update-incompatibility-generic-notice

```diff
@@ -0,0 +1,4 @@
+Significance: minor
+Type: enhancement
+
+Update the generic incompatibility notice message for the Cart and Checkout page.
```

Comment:
do we need a changelog entry here too? 🤔

Reply:
@wavvves Sorry for answering this question delayed. The changelog entry in `/plugins/woocommerce/` is required.